data/geany.glade

```diff
@@ -6321,6 +6321,7 @@
               <object class="GtkMenuBar" id="menubar1">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
+                <signal name="deactivate" handler="on_menubar1_deactivate" after="yes" swapped="no"/>
```

Comment:
I'm not sure about "deselect" too. Probably yes, but it feels not safe to rely on "activate"-"deselect". But if we change "activate" to "select", then it'll be OK, IMHO.